.pre-commit-hooks.yaml

```
@@ -0,0 +1,15 @@
+- id: pylint-ignore
+  name: pylint
```

I think to avoid confusion, it would be better to name this just as the command is named.

```suggestion
  name: pylint-ignore
```

---

Fair enough!